tests/integration-tests/processors/processors-test.js

```diff
@@ -186,6 +190,113 @@ describe('Processors api', function() {
                 createProcessorResponse.statusCode.should.eql(201);
             });
         });
+        describe('PUT /v1/processors/{processor_id}', function() {
+            const baseURL = 'http://cloud.com';
+            const firstFileUri = '/files/888';
+            const secondFileUri = '/files/999';
+            const fileDownloadProcessor = {
+                name: 'predator-file',
+                type: PROCESSOR_TYPE_FILE_DOWNLOAD,
+                description: 'some desc'
+            };
+            it('update a raw_javascript type processor', async function() {
+                const processor = generateRawJSProcessor('predator');
+                const createResponse = await requestSender.createProcessor(processor, validHeaders);
+                should(createResponse.statusCode).equal(201);
+                const processorId = createResponse.body.id;
+
+                processor.javascript = 'module.exports.add = (a,b) => a + b;';
+                processor.description = 'add two numbers';
+                const updateResponse = await requestSender.updateProcessor(processorId, processor);
+                should(updateResponse.statusCode).equal(200);
+                should(updateResponse.body.javascript).equal(processor.javascript);
+                should(updateResponse.body.description).equal(processor.description);
+            });
+            it('update a file_download type processor - change to different file_url', async function() {
+                const processor = {
+                    name: 'predator-file',
+                    type: PROCESSOR_TYPE_FILE_DOWNLOAD,
+                    description: 'some desc',
+                    file_url: baseURL + firstFileUri
+                };
+                nock(baseURL).get(firstFileUri).reply(200, 'module.exports = 9');
+                const createResponse = await requestSender.createProcessor(processor, validHeaders);
+                should(createResponse.statusCode).equal(201);
+                const processorId = createResponse.body.id;
+
+                const updatedFileJSContent = 'module.exports = () => 1';
+                processor.file_url = baseURL + secondFileUri;
+                nock(baseURL).get(secondFileUri).reply(200, updatedFileJSContent);
+                const updateResponse = await requestSender.updateProcessor(processorId, processor, true);
+                should(updateResponse.statusCode).equal(200);
+                should(updateResponse.body.file_url).equal(processor.file_url);
+                should(updateResponse.body.javascript).equal(updatedFileJSContent);
+            });
+            it('update a file_download type processor - change to different file_url', async function() {
```

Comment:
seems like the same test as the one on line 215

Reply:
true, deleting it right now